runtime/lua/vim/_buf.lua

```diff
@@ -0,0 +1,21 @@
+-- TODO: move to _defaults.lua once it is possible to assign a Lua function to options #25672
```

Comment:
let's add this todo on each of the functions. since `_buf.lua` might be extended for other purposes later, having this top-level TODO would be confusing

Reply:
Done